helm-chart/kuberay-operator/values.yaml

```diff
@@ -17,6 +17,9 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: "kuberay-operator"
 
+scc:
```

Comment:
I'd removed the Helm related changes as suggested by @anishasthana

Reply:
Yea, these were added for consistency as I believe the CI tests make use of helm for deployment? However, as these tests also run on kind there is no need for the SCC anyway.